app/api-v1/controllers/Recipe/helpers.js

```diff
@@ -1,10 +1,12 @@
 const mapRecipeData = (data) => ({
+  externalId: { type: 'LITERAL', value: data.external_id },
   name: { type: 'LITERAL', value: data.name },
   material: { type: 'LITERAL', value: data.material },
   alloy: { type: 'LITERAL', value: data.alloy },
-  price: { type: 'LITERAL', value: data.price },
-  requiredCerts: { type: 'LITERAL', value: data.required_certs },
-  supplier: { type: 'LITERAL', value: data.supplier },
+  requiredCerts: { type: 'FILE', value: 'required_certs.json' },
+  transactionId: { type: 'LITERAL', value: data.transaction.id },
```

Comment:
So we can match up the recipe in the blockchain watcher?

Reply:
yes!